src/lbaf/Model/lbsPhase.py

```diff
@@ -494,12 +498,16 @@ def populate_from_specification(self, spec: PhaseSpecification, multiple_sharing
             "type": "SendRecv",
             "to": {
                 "type": "object",
-                "id": comm_spec["to"]
+                "seq_id": comm_spec["to"],
```

Comment:
Same here; the identifier key (either `id` or `seq_id`) needs to be flexible and based on the input JSON

Reply:
Ok @cwschilly there was some misunderstanding about `Use seq_id instead of id for objects/tasks (since id now means that the value is encoded)` from my side. 
I understand that LBAF should be finally be able to use either `id` or `seq_id`. This will impact the whole load balancing login as in algorithms. For example this kind of logic (BruteForceAlgorithm) will have to be modified as well
```python
object_id = objects[i].get_id()
for o in r_src.get_objects():
  if o.get_id() == object_id:
```
We have now to take care about which of `id` and `seq_id` is defined.
I will work on that.

Reply:
Done